frontend/src/components/StockNews/StockNews.jsx

```diff
@@ -87,7 +87,7 @@ export default function StockNews({ stocks }) {
             {stockNews?.length > articlesToShow ? (
                 <Button
                     variant="outline-primary"
-                    onClick={() => setArticlesToShow(articlesToShow + 5)}
+                    onClick={() => setArticlesToShow((prev) => prev + 3)}
```

Comment:
This 3 is a magic number. Let's store it in a constant and use such constant here.